### ISSUE TITLE
[a11y] Address contrast-comparison violations in wcag2aa report for espresso 

### ISSUE THREAD

**Issue by unlikelyzero** [nasa/openmct#7304]
Now that https://github.com/nasa/openmct/pull/7047 is complete, we've disabled the contrast-comparison ruleset to address in future iterations.

This ticket captures the work necessary to disable the rule in our tests without necessary addressing every violation in the snow theme

**Comment by charlesh88** [comment]
WIP pushed to branch `fix-contrast`.
